core/src/test/java/org/apache/iceberg/hadoop/TestStaticTable.java

```diff
@@ -88,13 +87,13 @@ public void testHasSameProperties() {
     table.newAppend().appendFile(FILE_B).commit();
     table.newOverwrite().deleteFile(FILE_B).addFile(FILE_C).commit();
     Table staticTable = getStaticTable();
-    Assert.assertTrue("Same history?", table.history().containsAll(staticTable.history()));
-    Assert.assertTrue(
-        "Same snapshot?",
-        table.currentSnapshot().snapshotId() == staticTable.currentSnapshot().snapshotId());
-    Assert.assertTrue(
-        "Same properties?",
-        Maps.difference(table.properties(), staticTable.properties()).areEqual());
+    Assertions.assertThat(table.history()).as("Same history?").containsAll(staticTable.history());
+    Assertions.assertThat(table.currentSnapshot().snapshotId())
+        .as("Same snapshot?")
+        .isEqualTo(staticTable.currentSnapshot().snapshotId());
+    Assertions.assertThat(Maps.difference(table.properties(), staticTable.properties()).areEqual())
```

Comment:
can we express that with an AssertJ assertion rather than using isTrue?

Reply:
I've modified that code based on what you suggested.